docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+| ---------------------- | ---------------------------------------------- | ------------- | ----------------------- | ---------------------- |
+| \*.ubam_qc_metrics.tgz | Generated by Picard CollectQualityYieldMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Alignment QC
+
+Files containing quality control metrics for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
+
+### Duplicates Metrics
+
+Data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+
+#### File Types
+
+| Filename Pattern          | Description                                | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
```

Comment:
is there anything else to say to define this?

Reply:
need to talk tok bioinfo

Reply:
confirmed should be enough with christina